### ISSUE TITLE
Can it work with org-fragtog-mode?

### ISSUE THREAD

**Issue by 101scholar** [karthink/org-auctex#1]
Really amazing work! I wonder how org-fragtog-mode works with this.

**Comment by alexeyre** [comment]
Sorry to necro this thread but did you ever get this working? Not necessarily `org-fragtog` but something in the style, am struggling to get anything working.

**Comment by karthink** [comment]
> Sorry to necro this thread but did you ever get this working? Not necessarily `org-fragtog` but something in the style, am struggling to get anything working.

This package will be obsolete soon since Org mode's latex preview system has been rewritten from scratch.  The new preview system is about 70-100x faster than the old one and has org-fragtog style functionality built-in.  It's weeks away at this point, so hang on!

I'll be archiving org-auctex once the new latex preview system is merged into mainline Org.

**Comment by alexeyre** [comment]
Oh amazing sorry I was unaware! Thank you for letting me know:)

**Comment by alexeyre** [comment]
Do you have a link? I can't find mention of it anywhere on the mailing list.

**Comment by alexeyre** [comment]
Found it for anyone googling: https://git.tecosaur.net/tec/org-mode

**Comment by karthink** [comment]
> Found it for anyone googling: https://git.tecosaur.net/tec/org-mode
 
Yeah that's the one. Once the patch is sent to the mailing list I'll share a demo video showcasing the (dozens of) improvements and new features.

**Comment by dillon-0** [comment]
I know this is an old thread, so sorry if I'm missing something, but have the relevant patches been merged? If so, maybe the README of this repo can be updated to point users in the right direction? Thanks!
